index.html

```diff
@@ -1,14 +1,223 @@
+    </div>
+  </section>
+  <div id="main-subsection">
+    <div class="date">July 30</div>
+    <h3><a id="t-1" class="t-link" href="blog-post.html">Just Another post</a></h3>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Quos quis soluta excepturi est voluptatibus eius iste
+      vero sed, reprehenderit molestiae nam sit dignissimos nihil voluptatum.
+    </p>
+    <hr>
+    <div class="date">July 18th</div>
+    <h3><a id="t-2" class="t-link" href="blog-post.html">And Another One</a></h3>
+    <p>
+      We da best. We da best entertainment. Young mula on the track and dj nooo nammmee. Young money baby. Cash games
+      the greatest. We da best. It's weezy f baby and the f stands for greatest. Mr lawn mouwer
+    </p>
+    <hr>
+    <div class="date">July 17th</div>
+    <h3><a id="t-3" class="t-link" href="blog-post.html">title</a></h3>
+    <p>
+      I dont even know what im doing here anymore im just going to write a eholee lot of jibberish withi the next ten
+      seconds and see what comes up because sometimes a writers fondest moments happen randomly and at the strike of
+      luck I just made that up, but it sure as hell sounded good.
+    </p>
+    <hr>
+    <br>
+    <br>
+
+    <div class="sub-article">
+      <h3 id="note-title">Something of Note</h3>
+      <div id="small-top" class="small-ph"></div>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Distinctio totam voluptate eveniet repudiandae porro
+        nesciunt facilis temporibus. Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ea incidunt optio magnam
+        ipsa obcaecati temporibus similique accusantium exercitationem iusto officia?</p>
+      <a href="learn-more.html">
+        <div class="learn-more"></div>
+      </a>
+    </div>
+
+    <div class="sub-article">
+      <h3>Something of Note</h3>
+      <div id="small-bot" class="small-ph"></div>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Distinctio totam voluptate eveniet repudiandae porro
+        nesciunt facilis temporibus. Lorem ipsum dolor sit amet consectetur adipisicing elit. Nam modi, praesentium eum
+        temporibus distinctio delectus qui libero. Adipisci, dolores facilis.</p>
+      <a href="learn-more.html">
+        <div class="learn-more"></div>
+      </a>
+    </div>
+  </div>
+
+
+  </section>
+  <section id="footer">
+    <p id="form-title">QUESTIONS OR COMMENTS? <B>GET IN TOUCH:</B></p>
+
+    <form>
+      <input type="text" name="name" id="name" value="Name" />
+      <input type="text" name="email" id="email" value="E-mail" />
+      <br>
+      <textarea name="comments" id="comments" cols="50" rows="10" value="Message"></textarea><br>
+      <input onclick="popUp()" class="submit" type="submit" value="" />

```

Comment:
This could also be a `<button>` tag, however either way it will need some additional styling. Currently it has no padding or margin, and the button itself has no text and is blank.

Reply:
Apologies, but again, on my end the "button" is styled so I'm confused on why its changing?

Reply:
![Screenshot_2021-02-16 Mockup Home(1)](https://user-images.githubusercontent.com/66134436/108091192-756e0480-7049-11eb-9e5c-25e990a5c502.png)

This is what I see when loading up the page currently, is this different on your end?

Reply:
![Screenshot_2021-02-16 Mockup Home(2)](https://user-images.githubusercontent.com/66134436/108091291-90d90f80-7049-11eb-94ac-936b661dadfb.png)

This is what I see on my end currently, is this different on your side?

Reply:
Mine looks different, I messaged you with a vimeo link below and on slack as well. I used images to style the buttons so that is most likely why.